KIPs/kip-34.md

```diff
@@ -390,7 +393,8 @@ Each keyring class uses the `PrivateKey` class, which has one private key as a m
 | Method | Description |
 | ----------- | ----------- |
 | getPublicKey(): List&#60;String&#62; | Returns the public key strings. |
-| getKeyByRole(role): List&#60;PrivateKey&#62; | Returns the keys specified by the role. |
+| getPublicKey(compressed: Boolean): String | Returns the compressed public key strings if compressed is true. It returns the uncompressed public key strings otherwise. |
+| getKeyByRole(role: int): List&#60;PrivateKey&#62; | Returns the keys specified by the role. |
```

Comment:
```suggestion
| getKeyByRole(role: int): List&#60;PrivateKey&#62; | Returns the keys specified by the role. MultipleKeyring returns the same keys since it does not have role. |
```